core/templates/components/question-directives/question-editor/question-editor.component.ts

```diff
@@ -166,6 +166,7 @@ angular.module('oppia').component('questionEditor', {
       };
 
       ctrl.saveHints = function(displayedValue) {
+        $rootScope.$applyAsync();
```

Comment:
Should we move this inside the callback passed to `_updateQuestion` since that is conceptually where the "update" actually happens before we want to refresh the view?

Reply:
done